content/en/docs/concepts/packages/_index.md

```diff
@@ -77,8 +76,8 @@ within the module root:
 
     cue mod init [module name]
 
-The module name is required if a package within the module needs to
-import another package within the module.
+The module name is required only if a package within the module needs
```

Comment:
Pinging @mpvl for thoughts here on whether this language change now shifts the semantics to `iff` rather than `if`.

Reply:
Can we get feedback on whether this change is still controversial?